components/ee/agent-smith/pkg/agent/agent.go

```diff
@@ -240,7 +249,10 @@ type GradedInfringementKind string
 
 // GradeKind produces a graded infringement kind from severity and kind
 func GradeKind(kind InfringementKind, severity InfringementSeverity) GradedInfringementKind {
-	return GradedInfringementKind(fmt.Sprintf("%s %s", severity, kind))
+	if len(severity) > 0 {
```

Comment:
> I'd argue for swapping the code paths to highlight what's the regular case.

Can you explain a bit more what do you mean?
I don't think I understand what is your suggestion here

Reply:
Can you explain a bit more? I wasn't able to understand

Reply:
`GradeKind` is more likely to be called with `len(severity) > 0` than not, hence `len(severity) > 0` is the happy path of this function and should be left aligned. Put differently, `len(severity) == 0` is the exception to the rule.

Frankly, the reason I even commented is because I thought `len(severity) == 0` never ordinarily happens, but upon looking through the code again I found that [`InfringementSeverityAudit`](https://github.com/gitpod-io/gitpod/blob/7ebfcf518bf50f6ef59df28f36ac778e5462cd74/components/ee/agent-smith/pkg/agent/agent.go#L228) indeed is an empty string. The above comment still stands, although has become even more nittier than it was before :/

Reply:
ah ok understood

Reply:
should be done now